frontend/src/views/SettingWorkspaceDebugLogs.vue

```diff
@@ -0,0 +1,124 @@
+<template>
+  <div class="space-y-4">
+    <div v-if="debugLogList.length > 0">
+      <div class="flex flex-row justify-between items-center textinfolabel">
+        {{
+          $t("debug-logs.count-of-logs", {
+            count: debugLogList.length,
+          })
+        }}
+        <button
+          class="group btn-normal items-center !border-0 !bg-accent !text-white hover:!bg-indigo-500"
+          @click="handleExport"
+        >
+          <heroicons-outline:document-arrow-down class="h-4 w-4 mr-1" />
+          {{ $t("debug-logs.table.operation.export") }}
+        </button>
+      </div>
+      <DebugLogsTable
+        :debug-log-list="debugLogList"
+        @view-detail="
+          (log) => {
+            state.modalContent = log
+            state.showModal = true;
+            dialog!.open();
+          }
+        "
+      />
+      <BBDialog
+        ref="dialog"
+        :title="$t('debug-logs.debug-logs-detail')"
+        :negative-text="$t('common.close')"
+        :positive-text="$t('debug-logs.table.operation.copy')"
+        data-label="bb-migration-mode-dialog"
+        @before-positive-click="handleCopy"
+      >
+        <div class="w-192 font-mono">
+          <dl>
+            <dd
+              v-for="(value, key) in state.modalContent"
+              :key="key"
+              class="flex items-start text-sm md:mr-4 mb-1"
+            >
+              <NGrid x-gap="20" :cols="20">
+                <NGi span="3">
+                  <span class="textlabel whitespace-nowrap">{{
+                    logKeyMap[key]
+                  }}</span
+                  ><span class="mr-1">:</span>
+                </NGi>
+                <NGi span="17">
+                  <span v-if="value !== ''">
+                    {{
+                      key === "RecordTs"
+                        ? dayjs.unix(value as number).format("YYYY-MM-DD HH:mm:ss Z")
+                        : value
+                    }}
+                  </span>
+                  <span v-else class="italic text-gray-500">
+                    {{ $t("debug-logs.table.empty") }}
+                  </span>
+                </NGi>
+              </NGrid>
+            </dd>
+          </dl>
+        </div>
+      </BBDialog>
+    </div>
+    <DebugLogsEmptyView v-else />
```

Comment:
Consider making all DebugLogs and debug-logs to DebugLog and debug-log. We should either use debug-logs or debug-log consistency if possible.